crates/holochain_zome_types/src/warrant.rs

```diff
@@ -21,32 +28,41 @@ use crate::signature::Signed;
     derive_more::Deref,
 )]
 pub struct Warrant {
-    /// The self-proving part of the warrant containing evidence of bad behavior
+    /// The self-proving part of the warrant containing evidence of bad behavior.
     #[deref]
     pub proof: WarrantProof,
-    /// The author of the warrant
+    /// The author of the warrant.
     pub author: AgentPubKey,
-    /// Time when the warrant was issued
+    /// Time when the warrant was issued.
```

Comment:
Not to nitpick but I don't think these are complete sentences :smile:

Reply:
I know, it's just a pattern that I like. Better than not ending complete sentences with a full stop.

Reply:
> Better than not ending complete sentences with a full stop.
If you say so :smile:

Reply:
In the code, I mean